// expo/build/ExpoLazy.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "apisAreAvailable", {
  enumerable: true,
  get: function () {
    return _apisAreAvailable.default;
  }
});
Object.defineProperty(exports, "AppLoading", {
  enumerable: true,
  get: function () {
    return _AppLoading.default;
  }
});
Object.defineProperty(exports, "AuthSession", {
  enumerable: true,
  get: function () {
    return _AuthSession.default;
  }
});
Object.defineProperty(exports, "DangerZone", {
  enumerable: true,
  get: function () {
    return _DangerZone.default;
  }
});
Object.defineProperty(exports, "Linking", {
  enumerable: true,
  get: function () {
    return _Linking.default;
  }
});
Object.defineProperty(exports, "Notifications", {
  enumerable: true,
  get: function () {
    return _Notifications.default;
  }
});
Object.defineProperty(exports, "Pedometer", {
  enumerable: true,
  get: function () {
    return _Pedometer.default;
  }
});
Object.defineProperty(exports, "registerRootComponent", {
  enumerable: true,
  get: function () {
    return _registerRootComponent.default;
  }
});
Object.defineProperty(exports, "Accelerometer", {
  enumerable: true,
  get: function () {
    return _deprecated.Accelerometer;
  }
});
Object.defineProperty(exports, "AdMobBanner", {
  enumerable: true,
  get: function () {
    return _deprecated.AdMobBanner;
  }
});
Object.defineProperty(exports, "AdMobInterstitial", {
  enumerable: true,
  get: function () {
    return _deprecated.AdMobInterstitial;
  }
});
Object.defineProperty(exports, "AdMobRewarded", {
  enumerable: true,
  get: function () {
    return _deprecated.AdMobRewarded;
  }
});
Object.defineProperty(exports, "Animated", {
  enumerable: true,
  get: function () {
    return _deprecated.Animated;
  }
});
Object.defineProperty(exports, "Amplitude", {
  enumerable: true,
  get: function () {
    return _deprecated.Amplitude;
  }
});
Object.defineProperty(exports, "AppAuth", {
  enumerable: true,
  get: function () {
    return _deprecated.AppAuth;
  }
});
Object.defineProperty(exports, "Asset", {
  enumerable: true,
  get: function () {
    return _deprecated.Asset;
  }
});
Object.defineProperty(exports, "Audio", {
  enumerable: true,
  get: function () {
    return _deprecated.Audio;
  }
});
Object.defineProperty(exports, "BackgroundFetch", {
  enumerable: true,
  get: function () {
    return _deprecated.BackgroundFetch;
  }
});
Object.defineProperty(exports, "BarCodeScanner", {
  enumerable: true,
  get: function () {
    return _deprecated.BarCodeScanner;
  }
});
Object.defineProperty(exports, "Barometer", {
  enumerable: true,
  get: function () {
    return _deprecated.Barometer;
  }
});
Object.defineProperty(exports, "BlurView", {
  enumerable: true,
  get: function () {
    return _deprecated.BlurView;
  }
});
Object.defineProperty(exports, "Brightness", {
  enumerable: true,
  get: function () {
    return _deprecated.Brightness;
  }
});
Object.defineProperty(exports, "Calendar", {
  enumerable: true,
  get: function () {
    return _deprecated.Calendar;
  }
});
Object.defineProperty(exports, "Camera", {
  enumerable: true,
  get: function () {
    return _deprecated.Camera;
  }
});
Object.defineProperty(exports, "Constants", {
  enumerable: true,
  get: function () {
    return _deprecated.Constants;
  }
});
Object.defineProperty(exports, "Contacts", {
  enumerable: true,
  get: function () {
    return _deprecated.Contacts;
  }
});
Object.defineProperty(exports, "Crypto", {
  enumerable: true,
  get: function () {
    return _deprecated.Crypto;
  }
});
Object.defineProperty(exports, "DocumentPicker", {
  enumerable: true,
  get: function () {
    return _deprecated.DocumentPicker;
  }
});
Object.defineProperty(exports, "Easing", {
  enumerable: true,
  get: function () {
    return _deprecated.Easing;
  }
});
Object.defineProperty(exports, "Facebook", {
  enumerable: true,
  get: function () {
    return _deprecated.Facebook;
  }
});
Object.defineProperty(exports, "FacebookAds", {
  enumerable: true,
  get: function () {
    return _deprecated.FacebookAds;
  }
});
Object.defineProperty(exports, "FaceDetector", {
  enumerable: true,
  get: function () {
    return _deprecated.FaceDetector;
  }
});
Object.defineProperty(exports, "FileSystem", {
  enumerable: true,
  get: function () {
    return _deprecated.FileSystem;
  }
});
Object.defineProperty(exports, "Font", {
  enumerable: true,
  get: function () {
    return _deprecated.Font;
  }
});
Object.defineProperty(exports, "GestureHandler", {
  enumerable: true,
  get: function () {
    return _deprecated.GestureHandler;
  }
});
Object.defineProperty(exports, "GL", {
  enumerable: true,
  get: function () {
    return _deprecated.GL;
  }
});
Object.defineProperty(exports, "GLView", {
  enumerable: true,
  get: function () {
    return _deprecated.GLView;
  }
});
Object.defineProperty(exports, "GoogleSignIn", {
  enumerable: true,
  get: function () {
    return _deprecated.GoogleSignIn;
  }
});
Object.defineProperty(exports, "Gyroscope", {
  enumerable: true,
  get: function () {
    return _deprecated.Gyroscope;
  }
});
Object.defineProperty(exports, "Haptic", {
  enumerable: true,
  get: function () {
    return _deprecated.Haptic;
  }
});
Object.defineProperty(exports, "Haptics", {
  enumerable: true,
  get: function () {
    return _deprecated.Haptics;
  }
});
Object.defineProperty(exports, "Icon", {
  enumerable: true,
  get: function () {
    return _deprecated.Icon;
  }
});
Object.defineProperty(exports, "ImageManipulator", {
  enumerable: true,
  get: function () {
    return _deprecated.ImageManipulator;
  }
});
Object.defineProperty(exports, "ImagePicker", {
  enumerable: true,
  get: function () {
    return _deprecated.ImagePicker;
  }
});
Object.defineProperty(exports, "IntentLauncher", {
  enumerable: true,
  get: function () {
    return _deprecated.IntentLauncher;
  }
});
Object.defineProperty(exports, "IntentLauncherAndroid", {
  enumerable: true,
  get: function () {
    return _deprecated.IntentLauncherAndroid;
  }
});
Object.defineProperty(exports, "KeepAwake", {
  enumerable: true,
  get: function () {
    return _deprecated.KeepAwake;
  }
});
Object.defineProperty(exports, "LinearGradient", {
  enumerable: true,
  get: function () {
    return _deprecated.LinearGradient;
  }
});
Object.defineProperty(exports, "LocalAuthentication", {
  enumerable: true,
  get: function () {
    return _deprecated.LocalAuthentication;
  }
});
Object.defineProperty(exports, "Localization", {
  enumerable: true,
  get: function () {
    return _deprecated.Localization;
  }
});
Object.defineProperty(exports, "Location", {
  enumerable: true,
  get: function () {
    return _deprecated.Location;
  }
});
Object.defineProperty(exports, "Magnetometer", {
  enumerable: true,
  get: function () {
    return _deprecated.Magnetometer;
  }
});
Object.defineProperty(exports, "MagnetometerUncalibrated", {
  enumerable: true,
  get: function () {
    return _deprecated.MagnetometerUncalibrated;
  }
});
Object.defineProperty(exports, "MailComposer", {
  enumerable: true,
  get: function () {
    return _deprecated.MailComposer;
  }
});
Object.defineProperty(exports, "MapView", {
  enumerable: true,
  get: function () {
    return _deprecated.MapView;
  }
});
Object.defineProperty(exports, "MediaLibrary", {
  enumerable: true,
  get: function () {
    return _deprecated.MediaLibrary;
  }
});
Object.defineProperty(exports, "Permissions", {
  enumerable: true,
  get: function () {
    return _deprecated.Permissions;
  }
});
Object.defineProperty(exports, "Print", {
  enumerable: true,
  get: function () {
    return _deprecated.Print;
  }
});
Object.defineProperty(exports, "PublisherBanner", {
  enumerable: true,
  get: function () {
    return _deprecated.PublisherBanner;
  }
});
Object.defineProperty(exports, "Random", {
  enumerable: true,
  get: function () {
    return _deprecated.Random;
  }
});
Object.defineProperty(exports, "SecureStore", {
  enumerable: true,
  get: function () {
    return _deprecated.SecureStore;
  }
});
Object.defineProperty(exports, "Segment", {
  enumerable: true,
  get: function () {
    return _deprecated.Segment;
  }
});
Object.defineProperty(exports, "Sensors", {
  enumerable: true,
  get: function () {
    return _deprecated.Sensors;
  }
});
Object.defineProperty(exports, "Sharing", {
  enumerable: true,
  get: function () {
    return _deprecated.Sharing;
  }
});
Object.defineProperty(exports, "SMS", {
  enumerable: true,
  get: function () {
    return _deprecated.SMS;
  }
});
Object.defineProperty(exports, "Speech", {
  enumerable: true,
  get: function () {
    return _deprecated.Speech;
  }
});
Object.defineProperty(exports, "SQLite", {
  enumerable: true,
  get: function () {
    return _deprecated.SQLite;
  }
});
Object.defineProperty(exports, "Svg", {
  enumerable: true,
  get: function () {
    return _deprecated.Svg;
  }
});
Object.defineProperty(exports, "takeSnapshotAsync", {
  enumerable: true,
  get: function () {
    return _deprecated.takeSnapshotAsync;
  }
});
Object.defineProperty(exports, "TaskManager", {
  enumerable: true,
  get: function () {
    return _deprecated.TaskManager;
  }
});
Object.defineProperty(exports, "Transition", {
  enumerable: true,
  get: function () {
    return _deprecated.Transition;
  }
});
Object.defineProperty(exports, "Transitioning", {
  enumerable: true,
  get: function () {
    return _deprecated.Transitioning;
  }
});
Object.defineProperty(exports, "Video", {
  enumerable: true,
  get: function () {
    return _deprecated.Video;
  }
});
Object.defineProperty(exports, "WebBrowser", {
  enumerable: true,
  get: function () {
    return _deprecated.WebBrowser;
  }
});
Object.defineProperty(exports, "WebView", {
  enumerable: true,
  get: function () {
    return _deprecated.WebView;
  }
});
exports.SplashScreen = exports.Updates = exports.StoreReview = exports.ScreenOrientation = exports.Logs = exports.Google = exports.ErrorRecovery = exports.AR = void 0;

require("./Expo.fx");

var AR = _interopRequireWildcard(require("./AR"));

exports.AR = AR;

var ErrorRecovery = _interopRequireWildcard(require("./ErrorRecovery/ErrorRecovery"));

exports.ErrorRecovery = ErrorRecovery;

var Google = _interopRequireWildcard(require("./Google"));

exports.Google = Google;

var Logs = _interopRequireWildcard(require("./logs/Logs"));

exports.Logs = Logs;

var ScreenOrientation = _interopRequireWildcard(require("./ScreenOrientation/ScreenOrientation"));

exports.ScreenOrientation = ScreenOrientation;

var StoreReview = _interopRequireWildcard(require("./StoreReview/StoreReview"));

exports.StoreReview = StoreReview;

var Updates = _interopRequireWildcard(require("./Updates/Updates"));

exports.Updates = Updates;

var SplashScreen = _interopRequireWildcard(require("./launch/SplashScreen"));

exports.SplashScreen = SplashScreen;

var _apisAreAvailable = _interopRequireDefault(require("./apisAreAvailable"));

var _AppLoading = _interopRequireDefault(require("./launch/AppLoading"));

var _AuthSession = _interopRequireDefault(require("./AuthSession"));

var _DangerZone = _interopRequireDefault(require("./DangerZone"));

var _Linking = _interopRequireDefault(require("./Linking/Linking"));

var _Notifications = _interopRequireDefault(require("./Notifications/Notifications"));

var _Pedometer = _interopRequireDefault(require("./Pedometer"));

var _registerRootComponent = _interopRequireDefault(require("./launch/registerRootComponent"));

var _deprecated = require("./deprecated");

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

function _interopRequireWildcard(obj) { if (obj && obj.__esModule) { return obj; } else { var newObj = {}; if (obj != null) { for (var key in obj) { if (Object.prototype.hasOwnProperty.call(obj, key)) { var desc = Object.defineProperty && Object.getOwnPropertyDescriptor ? Object.getOwnPropertyDescriptor(obj, key) : {}; if (desc.get || desc.set) { Object.defineProperty(newObj, key, desc); } else { newObj[key] = obj[key]; } } } } newObj.default = obj; return newObj; } }

// @ts-ignore
if (module && module.exports) {
  if (global) {
    const globals = require('./globals'); // @ts-ignore


    global.__exponent = globals; // @ts-ignore

    global.__expo = globals; // @ts-ignore

    global.Expo = globals;
  }
}
